.github/workflows/accept-baselines-fix-lints.yml

```diff
@@ -0,0 +1,29 @@
+name: Accept Baselines and Fix Lints
+
+on:
+  workflow_dispatch: {}
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use node version 12
+      uses: actions/setup-node@v1
+      with:
+        node-version: 12
+        registry-url: https://registry.npmjs.org/
+
+    - name: Configure Git, Run Tests, Update Baselines, Apply Fixes
+      run: |
+        git config user.email "ts_bot@rcavanaugh.com"
```

Comment:
perfect, blame it on Ryan!

Reply:
Okay, side note, all of our actions do this. Two things

1. Do we have a Microsoft account we can use @RyanCavanaugh?
2. Is there a way to use the person who triggered the PR's email?

Reply:
1. typescriptbot@microsoft.com
2. I believe `${{github.actor}}@users.noreply.github.com` should do the trick

Reply:
> I believe `${{github.actor}}@users.noreply.github.com` should do the trick

I think if your account isn't old enough, you have to use something like a special user ID email.

https://docs.github.com/en/free-pro-team@latest/github/setting-up-and-managing-your-github-user-account/setting-your-commit-email-address#about-commit-email-addresses

> If you created your GitHub account after July 18, 2017, your GitHub-provided no-reply email address is a seven-digit ID number and your username in the form of `ID+username@users.noreply.github.com`. If you created your GitHub account prior to July 18, 2017, your GitHub-provided no-reply email address is your username in the form of `username@users.noreply.github.com`. You can get an ID-based GitHub-provided no-reply email address by selecting (or deselecting and reselecting) Keep my email address private in your email settings.